Index: target/classes/application.properties
===================================================================
diff --git a/target/classes/application.properties b/target/classes/application.properties
deleted file mode 100644
--- a/target/classes/application.properties	(revision 34f06beadc91656d67b3dc4ad89e97a77e5d790f)
+++ /dev/null	(revision 34f06beadc91656d67b3dc4ad89e97a77e5d790f)
@@ -1,8 +0,0 @@
-spring.application.name=crud
-spring.datasource.url=jdbc:postgresql://localhost:5432/crud
-spring.datasource.username=postgres
-spring.datasource.password=password
-spring.datasource.driver-class-name=org.postgresql.Driver
-spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.hibernate.ddl-auto=update
-
Index: .gitignore
===================================================================
diff --git a/.gitignore b/.gitignore
new file mode 100644
--- /dev/null	(date 1739279148476)
+++ b/.gitignore	(date 1739279148476)
@@ -0,0 +1,9 @@
+/.idea/dataSources/0455b189-6967-4a6a-895b-9814aa11b1e5.xml
+/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst
+/.idea/dataSources/0455b189-6967-4a6a-895b-9814aa11b1e5/storage_v2/_src_/database/crud.PrsuAA.meta
+/.idea/dataSources.local.xml
+/.idea/dataSources/0455b189-6967-4a6a-895b-9814aa11b1e5/storage_v2/_src_/database/crud.PrsuAA/schema/information_schema.FNRwLQ.meta
+/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst
+/.idea/misc.xml
+/.idea/dataSources/0455b189-6967-4a6a-895b-9814aa11b1e5/storage_v2/_src_/database/crud.PrsuAA/schema/pg_catalog.0S1ZNQ.meta
+/.idea/workspace.xml
diff --git a/target/classes/application.yml b/target/classes/application.yml
deleted file mode 100644
